src/services/auth-service.ts

```diff
@@ -18,13 +18,20 @@ export class AuthService {
     return !this.isUserSignedIn();
   }
 
-  async signInWithPassword(data: { email: string; password: string }, onSuccess: () => void, onError: (error: AuthError) => void = (error) => console.warn('Encountered an error when signing in.', error.cause)) {
+  async signInWithPassword(data: { email: string; password: string }, onSuccess: () => void, onError: (error: AuthError) => void = (error) => {
+    console.warn('Encountered an error when signing in.', error.cause);
+    alert(error.message);
```

Comment:
I don't think we want to fire off an `alert` here, we probably just want to handle the error with the `onError` callback that we pass into the function. Does that make sense?